### DISCUSSION TITLE
Custom sponsor donation is not working

### DISCUSSION THREAD

**Discussion by devpolo** [community/community#5386] [Projects and Issues]
# Hello there 👋

Looks like I'm not able to sponsor any project with a custom amount. The `Select` button is not clickable.

<img width="364" alt="1" src="https://user-images.githubusercontent.com/36532199/131546042-48177da1-73f2-4178-954f-672460e9f070.png">

## Exploration

Here is the message I get if I remove the `disabled` attribute.
Furthermore, no network activity to report.
```diff
- <button type="submit" disabled>Select</button>
+ <button type="submit">Select</button>
```
<img width="352" alt="Capture d’écran 2021-08-31 à 19 09 42" src="https://user-images.githubusercontent.com/36532199/131546351-13ce98fc-6611-42dd-9cfb-794a359730a3.png">

Hope it may help. 
Cheers

**Comment by BastLast** [comment]
hi, same issue here, can someone help ?
